### PR TITLE
fix: set username issue in address screen

### PR DESCRIPTION
**Pull request by Extheoisah**
This fix ensures that when the user sets his/her username, we query for an update is that when the user presses the back or home button, the `set address` modal is not rendered as we currently have it.

### CODE REVIEW

File: app/screens/galoy-address-screen/galoy-address-screen.tsx [hunk 1]
```diff
@@ -92,7 +92,10 @@ gql`
 export const GaloyAddressScreen = () => {
   const { LL } = useI18nContext()
   const isAuthed = useIsAuthed()
-  const { data } = useAddressScreenQuery({ fetchPolicy: "cache-first", skip: !isAuthed })
+  const { data } = useAddressScreenQuery({
+    fetchPolicy: "cache-and-network",
```
Comment (daviroo) [inline app/screens/galoy-address-screen/galoy-address-screen.tsx:96]: If we refetch the query do we need this change?

Comment (nicolasburtey) [reply app/screens/galoy-address-screen/galoy-address-screen.tsx:96]: probably not a bad idea to avoid potential stale data

Comment (Extheoisah) [reply app/screens/galoy-address-screen/galoy-address-screen.tsx]: Oops! It was the first option I tried. forgot to delete it. I've updated it now

Comment (Extheoisah) [reply app/screens/galoy-address-screen/galoy-address-screen.tsx]: we don't need `cache-and-network` since we refetch the query

**Review by daviroo — APPROVED** [review]

**Pull request merged**